### DISCUSSION TITLE
will LAMMPS work with DeePMD-kit offline package?

### DISCUSSION THREAD

**Discussion by chandagsin** [deepmodeling/deepmd-kit#2377] [Q&A] [answered]
Dear Developers

I got issues when compiling DeePMD-kit from source code but successfully installed it using offline package. I was wondering if LAMMPS plugin mode works OK with DeePMD-kit 2.1.5 offline package? or, is there a pre-built DeePMD-kit module available for LAMMPS built-in mode?

**Comment by njzjz** [comment, marked as answer]
If you use the LAMMPS inside the offline package, it should be okay. If you use an external LAMMPS program, you may need to ensure the LAMMPS and the plugin are compiled with the same ABI (e.g. [C++11 ABI](https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html)) and MPI.

**Reply by chandagsin** [reply]
Thx! your meant the recently-released DeePMD-kit v2.2.0 & v2.2.1? I tried v 2.1.5 but it doen't appear to be a LAMMPS inside the package?

**Reply by njzjz** [reply]
It's contained in all versions.

**Reply by chandagsin** [reply]
Thanks! How to load the LAMMPS module?
